charts/osdfir-infrastructure/README.md

```diff
@@ -0,0 +1,295 @@
+
+### OSDFIR Infrastructure persistence storage parameters
+
+| Name                       | Description                                                 | Value               |
+| -------------------------- | ----------------------------------------------------------- | ------------------- |
+| `persistence.enabled`      | Enables persistent volume storage for OSDFIR Infrastructure | `true`              |
+| `persistence.name`         | OSDFIR Infrastructure persistent volume name                | `osdfirvolume`      |
+| `persistence.size`         | OSDFIR Infrastructure persistent volume size                | `8Gi`               |
+| `persistence.storageClass` | PVC Storage Class for OSDFIR Infrastructure volume          | `""`                |
+| `persistence.accessModes`  | PVC Access Mode for the OSDFIR Infrastructure volume        | `["ReadWriteOnce"]` |
+
+### Timesketch Configuration
+
+| Name                                          | Description                                                                                                                           | Value                                                     |
+| --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------- |
+| `timesketch.enabled`                          | Enables the Timesketch deployment                                                                                                     | `true`                                                    |
+| `timesketch.image.repository`                 | Timesketch image repository                                                                                                           | `us-docker.pkg.dev/osdfir-registry/timesketch/timesketch` |
+| `timesketch.image.tag`                        | Overrides the image tag whose default is the chart appVersion                                                                         | `latest`                                                  |
+| `timesketch.config.override`                  | Overrides the default Timesketch configs to instead use a user specified directory if present on the root directory of the Helm chart | `configs/*`                                               |
+| `timesketch.config.createUser`                | Creates a default Timesketch user that can be used to login to Timesketch after deployment                                            | `true`                                                    |
+| `timesketch.frontend.resources.limits`        | The resources limits for the frontend container                                                                                       | `{}`                                                      |
```

Comment:
FYI, small nit, but it looks like there is a lot of trailing whitespace in this section.

Reply:
Ah yeah that was a result of the `readme-generator` not sure if there is a way to remove it, but somehow still formats it nicely when you view the rendered version. Cant look at more configuration options down the line